build.gradle.kts

```diff
@@ -24,7 +24,7 @@ buildscript {
     }
     dependencies {
         classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:${com.unciv.build.BuildConfig.kotlinVersion}")
-        classpath("com.android.tools.build:gradle:8.2.2")
+        classpath("com.android.tools.build:gradle:8.5.0")
```

Comment:
Please revert. Seems to be acting as a blocker (haven't checked the details behind it yet). Would be out of scope for the PR anyways

Reply:
Is that the problem? Android Studio offered me an update, I didn't think it would cause problems.

Reply:
I wasn't aware it mattered, so I just clicked on the update button. It looks like the problem has now been solved (I'm getting errors within Android studio, but not on Github).